pyxeoma/xeoma.py

```diff
@@ -74,30 +74,30 @@ def async_fetch_image_data(self, image_name, username, password):
             params['user'] = self.encode_user(username, password)
         else:
             params['user'] = ''
-        with aiohttp.ClientSession(cookies=cookies) as session:
-            resp = yield from session.get(
+        async with aiohttp.ClientSession(cookies=cookies) as session:
+            resp = await session.get(
                 '{}/{}.jpg'.format(self._base_url, image_name),
                 params=params
             )
             if resp.headers['Content-Type'] == 'image/jpeg':
-                data = yield from resp.read()
+                data = await resp.read()
             else:
                 data = None
         return data
 
     @asyncio.coroutine
-    def async_get_image_names(self):
+    async def async_get_image_names(self):
         """
             Parse web server camera view for camera image names
         """
 
         cookies = self.get_session_cookie()
         try:
             with aiohttp.ClientSession(cookies=cookies) as session:
```

Comment:
Missing `async`

Reply:
Actually don't worry about it. I'll take care of this when I create the release. Thanks again